tests/utils.js

```diff
@@ -691,7 +691,7 @@ module.exports = {
         method: 'POST',
         body: { docs: docs }
       });
-
+      
```

Comment:
```suggestion

```

There are whitespaces here that should be removed. Chances are your editor slipped them in there to "help".